### ISSUE TITLE
[BUG] <title>Issue running any eval using nexa

### ISSUE THREAD

**Issue by kzos** [NexaAI/nexa-sdk#375]
### Issue Description

2025-02-05:16:58:41,571 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,572 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,572 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,573 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,573 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,574 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,574 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
 20%|█▉        | 107/541 [00:00<00:00, 1063.17it/s]2025-02-05:16:58:41,575 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,575 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,577 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,577 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,578 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,578 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,578 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,579 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,579 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,580 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,581 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,581 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,581 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,582 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,582 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,582 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,583 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,583 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,584 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,584 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None
2025-02-05:16:58:41,584 ERROR    [nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False
2025-02-05:16:58:41,584 ERROR    [nexa_models.py:91] Invalid response for greedy_until. Response: None

[nexa_models.py:38] Unexpected error occured: logprobs is not supported for models created with logits_all=False

### Steps to Reproduce

nexa eval Qwen2-1.5B-Instruct:q4_0 --tasks ifeval
nexa eval Qwen2-1.5B-Instruct:q4_0 --tasks mmlu_pro

Getting error - logprobs is not suported for models created with logits_all=False 

### OS

Ubuntu 22.04

### Python Version

3.12

### Nexa SDK Version

0.0.9.8

### GPU (if using one)

4070

**Comment by MaokunZhang** [comment]
👋 Important Update: Nexa-SDK was fully rewritten on July 23, 2025, paving the way for exciting new features. This issue relates to a legacy version (v1) and has now been archived with the label legacy.

👉 Still experiencing this issue in the new version? Please open a new issue
🔗 Explore what's new in Nexa-SDK v2: [README](https://github.com/NexaAI/nexa-sdk)

Thanks for helping us make Nexa-SDK even better! 🚀✨
